c-svc23Vectors/math.c

```diff
@@ -34,3 +36,94 @@ int dotProductVectors(const int *input1, const int *input2, int length) {
 int dotProduct3dVectors(const int input1[3], const int input2[3]) {
     return dotProductVectors((const int*)input1, (const int*)input2, 3);
 }
+
+double getBiggestNumber(double axisArray[], int length){
+    double highestNumber = 0;
+    int negativeOrPositive = 1;
+    int i;
+    for (i = 0; i < TIME_SERIES_VECTORS_AMOUNT; i++)
+    {
+        double currentNumber = axisArray[i];
```

Comment:
```suggestion
        int currentNumber = abs(axisArray[i]);
```
Implementer en abs funktion.